docs/ABOUT.md

```diff
@@ -0,0 +1,7 @@
+Python is an exceptionally strong language for beginners because the code is highly readable and in many cases is as natural as reading a sentence in English. Code can be written and executed from the command line, in an interaction iPython session, or in a [Jupyter](http://jupyter.org) (iPython) notebook.
```

Comment:
Another reason Python is a fantastic language for beginners is that there are many excellent resources for it.